### PR TITLE
Added an algorithm course (Ready to merge)

### PR DESCRIPTION
**Pull request by Ajay-singh1**
## What does this PR do?
Add resource(s) 

### Description
Added an algorithm course in urdu
### Why is this valuable (or not)?
Yes
### How do we know it's really free?
Youtube
### For book lists, is it a book? For course lists, is it a course? etc.
course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by sohansai — COMMENTED** [review]
Replace '|' with `-` or  `\| `

**Comment by sohansai** [comment]
when you've made changes, please add (changes made) to the title of your PR

**Comment by Ajay-singh1** [comment]
@sohansai  @Thenlie Please Review my PR!

File: courses/free-courses-ur.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 
+* [Algorithms](#algorithms)
 * [Android](#android)
 * [C++](#cplusplus)
 * [C#](#csharp)
 * [HTML and CSS](#html-and-css)
 * [Python](#python)
 
 
+### <a id="algorithm"></a>Algorithms
```
Comment (eshellman) [inline courses/free-courses-ur.md:11]:
```suggestion
### Algorithms
```
you missed the `s`

**Pull request merged**